### ISSUE TITLE
 Etherchain Light visit bugs

### ISSUE THREAD

**Issue by cuu** [paritytech/parity#7999]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Which Parity version?**: Parity/v1.10.0-unstable-eff5cab-20180223/x86_64-linux-gnu/rustc1.24.0
> - **Which operating system?**:  Linux
> - **How installed?**: via installer  from source
> - **Are you fully synchronized?**:  yes
> - **Which network are you connected to?**: demo poa
> - **Did you try to restart the node?**:  yes

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._

I just build a demo poa chain followed wiki 

when use Etherchain Light
to visit
http://localhost:3000/account/0x1bd63497b8db7f5c73ee7ee9cca29dcd97abfc30

parity crashed

shows me 

```
stack backtrace:
   0:       0xe152949edc - backtrace::backtrace::trace::h9b029d0fdbfd6263
   1:       0xe152949582 - backtrace::capture::Backtrace::new::hc989ad19f674032e
   2:       0xe151dc1735 - panic_hook::panic_hook::h4ded59abfceb318d

Thread '<unnamed>' panicked at 'hash exists; qed', /checkout/src/libcore/option.rs:891
```
I use this command to start node1

parity --config node1.toml  --fat-db=on --tracing=on  --pruning=archive

**Comment by cuu** [comment]
parity git version hash: eff5cab

**Comment by gituser** [comment]
I've just updated to `v1.9.4` to check if that tracing bug has been fixed and got the same error:

```
2018-03-07 20:00:09  Verifier #1 INFO import  Imported #2789204 aab9…c848 (106 txs, 4.62 Mgas, 537.56 ms, 17.19 KiB)
2018-03-07 20:00:21  Verifier #2 INFO import  Imported #2789205 9893…d7e9 (95 txs, 4.70 Mgas, 1096.85 ms, 16.07 KiB)
2018-03-07 20:00:50  IO Worker #2 INFO import     3/50 peers    186 KiB chain  202 MiB db  0 bytes queue   21 KiB sync  RPC:  0 conn,  0 req/s,  75 µs
2018-03-07 20:00:53  IO Worker #3 INFO import     3/50 peers    186 KiB chain  202 MiB db  0 bytes queue   21 KiB sync  RPC:  0 conn,  0 req/s,  75 µs
2018-03-07 20:00:55  IO Worker #1 INFO import     3/50 peers    186 KiB chain  202 MiB db  0 bytes queue   21 KiB sync  RPC:  0 conn,  0 req/s,  75 µs
2018-03-07 20:00:58  IO Worker #0 INFO import     3/50 peers    186 KiB chain  202 MiB db  0 bytes queue   21 KiB sync  RPC:  0 conn,  0 req/s,  75 µs

====================

stack backtrace:
   0:     0x7f08babb507c - backtrace::backtrace::trace::h7fe757c5a8cb7109
   1:     0x7f08babb4722 - backtrace::capture::Backtrace::new::hd7f0642352dfc6dd
   2:     0x7f08ba292045 - panic_hook::panic_hook::h0724e12d078acbd2

Thread '<unnamed>' panicked at 'hash exists; qed', /checkout/src/libcore/option.rs:891
```

**Comment by ordian** [comment]
This issue (tracing) was fixed (#7979, #8066) and backported to stable 1.9.5 (#8133). 
Compiling parity multiple times to confirm this [wasn't fun](https://gist.github.com/ordian/29dbfa4080140dd26052566834ef6414).
